src/main/java/com/team4/testingsystem/services/impl/TestsServiceImpl.java

```diff
@@ -138,18 +149,55 @@ public void deassign(long id) {
 
     @Override
     public Test start(long id) {
+
         if (testsRepository.start(Instant.now(), id) == 0) {
             throw new TestNotFoundException();
         }
         Test test = testGeneratingService.formTest(getById(id));
+
+        test.setFinishTime(Instant.now().plusSeconds(42 * 60));
```

Comment:
we decided to use 42 munutes for a timer. For the test it still should be 40 mins?